sdk/storage/azure-storage-file-datalake/setup.py

```diff
@@ -94,7 +94,7 @@
     ]),
     install_requires=[
         "azure-core<2.0.0,>=1.10.0",
-        "msrest>=0.6.18",
+        "msrest>=0.6.21",
```

Comment:
this is a preview release, are we planning to update the development status in another pr?